File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_1/The solution/Python/Q_2.py
# Read numbers from user
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))

# Calculate the difference
diff = num1 - num2

# Check difference
if diff >= 0:
    print("The difference is: ", diff)
else:
    print("The difference is: ", diff * -1)

File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_1/The solution/Java/Q_2.java
package com.company;

import java.util.Scanner;

public class Q_2 {
    public static void main(String[] args){
        int num1, num2, diff;
        Scanner input = new Scanner(System.in);

        // Read numbers from user
        System.out.print("Enter first number: ");
        num1 = input.nextInt();
        System.out.print("Enter second number: ");
        num2 = input.nextInt();

        // Calculate the difference
        diff = num1 - num2;

        // Check difference
        if(diff >= 0)
            System.out.println("The difference is: " + diff);
        else
            System.out.println("The difference is: " + diff * -1); // // بعكس اشارة الفرقبضربة ف سالب واحد


    }
}
